civihr_default_theme/scss/components/_header.scss

```diff
@@ -1,8 +1,12 @@
+$chr-hr-admin-mobile-line-height: 50px;
```

Comment:
We do `mobile first` so it would be great if variables are named in an opposite way:

```css
$chr-hr-admin-line-height: 50px;
$chr-hr-admin-margin:      25px;
$chr-hr-admin-desktop-margin:             33px;
```

Also, I think there is a slight problem with self-explanation of vars names, I read `$chr-hr-admin-margin` as a margin of an admin. I think it lacks an explanation of `margin of what?`. Is it a margin of a button?